package.json

```diff
@@ -61,7 +61,7 @@
     "build:template": "node ./src/tools/yaml-import.js",
     "checkov": "checkov --quiet -f template.yaml --framework cloudformation --external-checks-git git@github.com:alphagov/di-devplatform-checkov-hook.git//src/gds_digitalidentity_checkovhook/custom_policies",
     "sam:build": "npm run build && npm run build:template && npm run checkov && sam build",
-    "sam:deploy": "sam deploy --stack-name di-btm-${ENV_NAME} --parameter-overrides Environment=${CONFIG_NAME:-dev} --s3-prefix ${ENV_NAME}",
+    "sam:deploy": "sam deploy --stack-name di-btm-${ENV_NAME} --parameter-overrides Environment=dev PrivateConfigStack=${CONFIG_NAME:-none} --s3-prefix ${ENV_NAME}",
```

Comment:
Will the README need to be updated as well to explain how to deploy config stacks now?

Reply:
The mechanism for the user stays the same, it should only require the CONFIG_NAME env var to be set, but I'll take another look and clarify if necessary.